src/Files.SourceGenerator/Utilities.cs

```diff
@@ -0,0 +1,290 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Files.SourceGenerator
+{
+	internal static class Utilities
+	{
+		internal const string AttributeNamespace = $"{nameof(Files)}.App.Attributes.";
+
+		internal static INamedTypeSymbol ObjectSymbol { get; set; } = null!;
+
+		#region Abstract Syntax Tree Generate
```

Comment:
We should avoid using directives as per the contributing guidelines

Reply:
Thanks for mentioning. I will fix it soon.